tests/yam/yast_modules/ntp_configuration.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
```

Comment:
you don't need intermediate folder 'yast_modules', we are trying to avoid big tree of folders under yam/ and it is not a YaST module.